lib/serializer-utils.js

```diff
@@ -21,6 +21,9 @@ module.exports = function (collectionName, payload, opts) {
   }
 
   function getType(str) {
+    if ( ! _.isUndefined(opts.pluralizeType) && ! opts.pluralizeType) {
+      return str;
+    }
     return inflection.pluralize(str);
   }
 
```

Comment:
What about configuring the default value of the pluralizeType when the serialization utils is instantiated?
Example:

```
module.exports = function (collectionName, payload, opts) {
  // Default value: true
  var pluralizeType = _.isUndefined(opts.pluralizeType) || opts.pluralizeType;
  ...
  ...

  function getType(str) {
    return pluralizeType ? inflection.pluralize(str) : str;
  }
```